src/CTA.Rules.Models/ProjectActions.cs

```diff
@@ -14,6 +14,7 @@ public ProjectActions()
             FileActions = new BlockingCollection<FileActions>();
             PackageActions = new BlockingCollection<PackageAction>();
             ProjectReferenceActions = new BlockingCollection<string>();
+            ProjectLevelActions = new List<ProjectLevelAction>();
```

Comment:
Does this get used anywhere for now?

Reply:
Yes, it is used elsewhere in the class but was not initialized.